test/performance/performance.go

```diff
@@ -133,3 +134,19 @@ func AddTrace(logf logging.FormatLogger, tName string, traceID string) {
 		logf("Cannot write to trace file: %v", err)
 	}
 }
+
+func resolveEndpoint(kubeClientset *pkgTest.KubeClient, domain string, resolvable bool, endpointOverride string) (string, error) {
```

Comment:
Should we refactor this to be exposed in knative.dev/pkg? The logic is exactly the same as in the spoofing client there.

Reply:
OK. I can refactor this and move to knative.dev/pkg.  Let's hold this PR now.
/hold

Reply:
this should be removed now.